src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -307,12 +307,6 @@
       "defaultValue": "false",
       "description": "Whether to turn off HTTPS. This option only applies if IndividualAuth or OrganizationalAuth are not being used."
     },
-    "NoHttps": {
```

Comment:
Is this part of this change? If so, how?

Reply:
There was a rebasing error and it got duplicated. Right above it is another NoHttps